docker-compose.yml

```
@@ -0,0 +1,21 @@
+
+services:
+  vue-game:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        BUILDKIT_INLINE_CACHE: 1
+      cache_from:
+        - your-registry/your-vue-app:latest
+    container_name: vue-game-container
+    ports:
+      - "8080:8080"
```

I notice that in the above Dockerfile, you are exposing port 80

```
EXPOSE 80
```

So, should this be port 80?
```suggestion
      - "80:80"
```

OR, 

Should the above Dockerfile `EXPOSE 8080`?

---

Ah yes this should be port 80, but not sure whether Ill go the nginx route for deployment over the express setup for our use case. I had prematurely put installed the nginx dependency thinking we may need to use it over the express setup for the AWS ECS. Ill probably revert back the express setup until Caleb figures out the AWS stuff then we can choose whether we want to go the express route or nginx route, we could do both but probably counter productive in our case.